pages/person/_itemId/index.vue

```diff
@@ -135,23 +87,34 @@ export default Vue.extend({
   mixins: [imageHelper, timeUtils],
   data() {
     return {
-      item: {} as BaseItemDto,
-      appearances: [] as BaseItemDto[],
-      backdropImageSource: '',
-      breakpoints: {
-        600: {
-          visibleSlides: 3
-        },
-        960: {
-          visibleSlides: 4
-        },
-        1264: {
-          visibleSlides: 6
+      loading: true,
+      swiperOptions: {
+        initialSlide: 0,
+        freeMode: this.$browser.isMobile(),
+        effect: 'slide',
+        navigation: {
+          nextEl: `.swiper-next`,
+          prevEl: `.swiper-prev`
         },
-        1904: {
-          visibleSlides: 6
+        slidesPerView: 3,
+        breakpoints: {
+          600: {
+            slidesPerView: 4
+          },
+          960: {
+            slidesPerView: 6
+          },
+          1264: {
+            slidesPerView: 6
+          },
+          1904: {
+            slidesPerView: 8
+          }
```

Comment:
Can't we move this to the swiper-section component as a default or to a mixin? We're repeating it unnecessarily imo.